config.js

```diff
@@ -95,6 +95,11 @@ var conf = convict({
       format: String,
       default: undefined
     },
+    verificationSigninUrl: {
```

Comment:
I have it on good authority (@philbooth) that the auth server endpoint to verify tokens is going to be called `/token/verify`. Can we call this `tokenVerifyUrl`?